docs/en/loading/s3.md

```diff
@@ -188,9 +188,9 @@ You may want to customize the table that you are inserting into, for example, th
 
 In this example, we are creating a table based on knowledge of how the table will be queried and the data in the Parquet file. The knowledge of the data in the Parquet file can be gained by querying the file directly in S3.
 
-- Since a query of the dataset in S3 indicates that the `Timestamp` column contains data that matches a VARBINARY data type, the column type is specified in the following DDL.
-- By querying the data in S3, you can find that there are no `NULL` values in the dataset, so the DDL does not set any columns as nullable.
-- Based on knowledge of the expected query types, the sort key and bucketing column are set to the column `UserID`. Your use case might be different for this data, so you might decide to use `ItemID` in addition to or instead of `UserID` for the sort key.
+- Since a query of the dataset in S3 indicates that the `Timestamp` column contains data that matches a VARCHAR data type, and StarRocks can cast from VARCHAR to DATETIME the data type is changed to DATETIME in the following DDL.
```

Comment:
and StarRocks can cast from VARCHAR to DATETIME the data type is changed to DATETIME in the following DDL. -- need a comma before "the data type"?